friends/src/Components/FriendsList.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import axios from 'axios';
+import styled from 'styled-components';
+
+const List = styled.div`
+    display:flex;
+    flex-direction:column;
+    align-items:center;
+`;
+const Card = styled.div`
+background-color: lightblue;
+width: 200px;
+margin: 30px 0;
+`;
+
+class FriendsList extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            friends: []
+        };
+    }
+
+    componentDidMount() {
+        axios
+            .get('http://localhost:5000/friends')
+            .then(res => {
```

Comment:
I would also recommend moving this use of `componentDidMount` up to `App.js` so that the `friends` object array can be passed down from the application state as a prop, with the ability to accept new friend entries and updates.